models/core/transfers.yml

```diff
@@ -0,0 +1,36 @@
+
+version: 2
+
+models:
+  - name: transfers
+    description: "Harmony transfer events"
+    columns:
+      - name: block_id
+        description: The block ID
+        tests:
+          - not_null
+      - name: tx_hash
+        description: The transaction hash
+        tests:
+        - not_null
+      - name: block_timestamp
+        description: The time the block was minted
+        tests:
+          - not_null
+      - name: chain_id
```

Comment:
don't think there's chain id in this table

Reply:
Deleted: https://github.com/MetricsDAO/harmony_dbt/pull/19/commits/030cae378684c4f3ab40863164cfc74a48b76eb1